test/language/statements/class/elements/private-class-field-on-frozen-objects.js

```diff
@@ -19,8 +19,8 @@ flags: [onlyStrict]
 ---*/
 
 class Test {
-  f = Object.freeze(this);
-  #g = "Test262";
+  f = this;
+  #g = (Object.freeze(this), "Test262");
```

Comment:
why the previous version is not correct?

Reply:
Because it fails to create `f` because the Object is already frozen.